client/verta/verta/_internal_utils/model_dependencies.py

```diff
@@ -27,17 +27,22 @@ def unwrap(func: Callable) -> Callable:
 def modules_in_function_body(func: Callable) -> Set[str]:
     """Return a set of all base modules called within the body of the provided function."""
     _func = unwrap(func)
-    _globals = [
+    _global_modules = {
         value.__name__.split('.')[0]  # strip off submodules and classes
         for key, value in inspect.getclosurevars(_func).globals.items()
         if isinstance(value, ModuleType)
-    ]
-    _non_locals = [
-        value.__name__.split('.')[0]  # strip off submodules and classes
-        for key, value in inspect.getclosurevars(_func).nonlocals.items()
-        if isinstance(value, ModuleType)
-    ]
-    return set(_globals + _non_locals)
+    }
+    _global_classes = {
```

Comment:
This variable name makes it sound like this set contains classes or class names, but it actually contains modules right?

Reply:
obsolete